react/ShelfItem.js

```diff
@@ -20,8 +21,9 @@ export default class ShelfItem extends Component {
       const [seller = { commertialOffer: { Price: 0, ListPrice: 0 } }] = path(['sellers'], sku) || []
       const [referenceId = { Value: '' }] = path(['referenceId'], sku) || []
       const [image = { imageUrl: '' }] = path(['images'], sku) || []
-      const unmixedImage = { ...image, imageUrl: image.imageUrl.replace(/^https?:/, '') }
-      normalizedProduct.sku = { ...sku, seller, referenceId, image: unmixedImage }
+      const resizedImage = changeImageUrlSize(toHttps(image.imageUrl), 250)
```

Comment:
This could default to 250*2, to take into account retina displays (which is the standard for pretty much every phone out there)